ServerCore/ServerCore.csproj

```diff
@@ -9,12 +9,20 @@
   <ItemGroup>
     <Compile Remove="DataModel\**" />
     <Compile Remove="Data\**" />
+    <Compile Remove="Pages\Manage\**" />
```

Comment:
Is this part intentional? If so why?

Reply:
it did it automatically. Removing it now. Tested and everything still works